etc/desi.json

```diff
@@ -6,6 +6,23 @@
     },
     "datachallenge":{
         "exclude":[],
+        "dc17a-twopct":{
+            "dc17a-twopct/[README|twopct.ecsv|dc17a-lite.tar.gz]":"dc17a-twopct/toplevel.tar",
+            "dc17a-twopct/batch/.*":"dc17a-twopct/batch.tar",
+            "dc17a-twopct/fiberassign/.*":"dc17a-twopct/fiberassign.tar",
+            "dc17a-twopct/spectro/redux/dc17a2/exposures/201908.*":"dc17a-twopct/spectro/redux/dc17a2/exposures/201908nn.tar",
```

Comment:
That is cleaner for the entry in desi.yaml, but results in ~50 GB tarballs which is 10x smaller than what HPSS is optimized for.  I was trying to group directories into the 500 GB - 1 TB range, but perhaps I was trying to hard to be HPSS-optimal instead of human-optimal.